bc/core/utils/messages.py

```diff
@@ -1,4 +1,26 @@
-MASTODON_POST_TEMPLATE = """New filing in {docket}: {desc}
+from dataclasses import dataclass
+
+
+@dataclass
+class MastodonTemplate:
+    number_of_links: int
+    character_count: int
+    str_template: str
+
+    def __len__(self):
+        """Returns the length of the template without the placeholders
+
+        All links count as 23 characters in mastodon no matter how long
+        they really are.
+        """
+        return self.number_of_links * 23 + self.character_count
+
+
+POST_TEMPLATE = MastodonTemplate(
+    number_of_links=2,
+    character_count=32,
+    str_template="""New filing in {docket}: {desc}
 
 PDF: {pdf_link}
-Docket: {docket_link}"""
+Docket: {docket_link}""",
+)
```

Comment:
Can we add the doc number here too, if there is one? Perhaps:

> New filing in {docket}
> Doc #xyz: {desc}
> 
> PDF: 
> Docket:

Reply:
Sure!

Reply:
The posts using the new template look like this:

![image](https://user-images.githubusercontent.com/55959657/219438158-7a19f2c3-dbc4-4df0-a637-c565acad693f.png)